.github/workflows/no-std.yaml

```diff
@@ -39,7 +39,7 @@ jobs:
           override: true
       - run: |
           cd ci/no-std-check
-          make check-panic-conflict
+          make check-no-std
```

Comment:
was this a typo from before?

Reply:
Not a typo I'd say. Renamed so for more clarity.